tests/test_packages.py

```diff
@@ -0,0 +1,99 @@
+import pathlib
+import tempfile
+import textwrap
+
+import pytest
+
+from lektor import packages
+
+
+@pytest.fixture
+def temp_dir():
+    with tempfile.TemporaryDirectory() as temp_dir:
+        yield pathlib.Path(temp_dir)
```

Comment:
Pytest already has a `tmpdir` fixture built-in. No need to reinvent it. That fixture is considered deprecated and you should use `tmp_path` instead (which returns `pathlib.Path()` objects).

Reply:
Oh, I didn't know that existed - thanks for the hint.